globalize.gemspec

```diff
@@ -17,6 +17,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'activerecord', '>= 4.2.0', '< 4.3'
   s.add_dependency 'activemodel', '>= 4.2.0', '< 4.3'
+  s.add_dependency 'request_store', '~> 1.1'
```

Comment:
Yes, you're right.
I'm used to defining it this way, because it gives me a hint about when I added that dependency. (it's more applicable in Gemfile). And to be honest, I've never even thought about specifying it your way.
I can change it, if you want.